broker.go

```diff
@@ -84,6 +99,23 @@ func (b *Broker) Open(conf *Config) error {
 
 		b.conf = conf
 
+		// Create or reuse the metrics shared between brokers
+		b.incomingByteRate = metrics.GetOrRegisterMeter("incoming-byte-rate", conf.MetricRegistry)
+		b.requestRate = metrics.GetOrRegisterMeter("request-rate", conf.MetricRegistry)
+		b.requestSize = getOrRegisterHistogram("request-size", conf.MetricRegistry)
+		b.outgoingByteRate = metrics.GetOrRegisterMeter("outgoing-byte-rate", conf.MetricRegistry)
+		b.responseRate = metrics.GetOrRegisterMeter("response-rate", conf.MetricRegistry)
+		b.responseSize = getOrRegisterHistogram("response-size", conf.MetricRegistry)
+		// Do not gather metrics for seeded broker (-1 id)
+		if b.id >= 0 {
```

Comment:
They do not provide the network related metrics for seed brokers either:
- https://github.com/apache/kafka/blob/d7de59a579af5ba4ecb1aec8fed84054f8b86443/clients/src/main/java/org/apache/kafka/common/network/Selector.java#L302
- https://github.com/apache/kafka/blob/d7de59a579af5ba4ecb1aec8fed84054f8b86443/clients/src/main/java/org/apache/kafka/common/network/Selector.java#L655
- https://github.com/apache/kafka/blob/d7de59a579af5ba4ecb1aec8fed84054f8b86443/clients/src/main/java/org/apache/kafka/common/Node.java#L20

I do not know why they chose not to but I found this comment on a JIRA ticket:
https://issues.apache.org/jira/browse/KAFKA-1251?focusedCommentId=13948165&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13948165